api/src/TestHelpers/BehatFixtures.php

```diff
@@ -0,0 +1,57 @@
+<?php declare(strict_types=1);
+
+
+namespace App\TestHelpers;
+
+use App\Entity\User;
+use Doctrine\Common\DataFixtures\Purger\ORMPurger;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+
+// Not extending AbstractDataFixture so we can use this in test runs rather commands
+class BehatFixtures
+{
+    private EntityManagerInterface $entityManager;
+    private array $fixtureParams;
+    private UserPasswordEncoderInterface $encoder;
+
+    public function __construct(
+        EntityManagerInterface $entityManager,
+        array $fixtureParams,
+        UserPasswordEncoderInterface $encoder
+    ) {
+        $this->entityManager = $entityManager;
+        $this->fixtureParams = $fixtureParams;
+        $this->encoder = $encoder;
+    }
+
+    public function loadFixtures()
+    {
+        $purger = new ORMPurger($this->entityManager);
+        $purger->purge();
+
+        // Add admin users
+        $adminUser = (new User())
+            ->setFirstname('Admin')
+            ->setLastname('User')
+            ->setEmail('admin@publicguardian.gov.uk')
```

Comment:
We probably want to have a pattern for fixtures relating to a particular test to avoid potential collisions in parallel runs and the DB vs ones that will never change.

Reply:
Great point - I'll add something in for that